exercises/exercise3.js

```diff
@@ -1,12 +1,419 @@
+    + '\n' + link;
+    },
+    isListening: true,
+    isCaseSensitive: false,
+  },
+  {
+    label: 'What are you designed to do',
+    prompt: 'What do you do then?',
+    handler: function() {
+      var name = 'Acacia filicifolia';
+      var link = 'https://en.wikipedia.org/wiki/Acacia_filicifolia';
+    return 'I help identify Native Australian plants... '
+    + '\nLike the common Native Fern-Leaved Wattle, aka ' + name 
+    + ' whihc is widespread across the New South Wales Tablelands...'
+    + '\n' + link;
+    },
+    isListening: true,
+    isCaseSensitive: false,
+  },
+  {
+    label: 'Bot explained script 0',
+    prompt: 'Okay, how do you do that?',
+    handler: function() {
+      return 'Tell me where you are by typing \'My location is...\' and I\'ll tell you about the Native plants in the area...'
+      + '\n(right now I\'m still bound to location only... but in the future I\'ll be able to identify plants based on things like petal shape, location, colour and seeds)';
+      }, 
+    isListening: true,
+    isCaseSensitive: false,
+  },
+  {
+    label: 'Bot explained script 1',
+    prompt: 'Okay, how do you do that then?',
+    handler: function() {
+      return 'ASL....'
+      + '\nJust kidding! That\'d be gross'
+      + '\nTell me where you are by typing \'Okay, my location is...\'and I\'ll tell you about the Native plants in the area...'
+      + '\n(right now I\'m still bound to location only... but in the future I\'ll be able to identify plants based on things like petal shape, location, colour and seeds)';
+      },
+    isListening: true,
+    isCaseSensitive: false,
+  },
+  {
+    label: 'Bot explained script 2',
+    prompt: 'How?',
+    handler: function() {
+      return 'Tell me where you are by typing \'My location is...\' and I\'ll tell you about the Native plants in the area...'
+      + '\n(right now I\'m still bound to location only... but in the future I\'ll be able to identify plants based on things like petal shape, location, colour and seeds)';
+    },
+    // isReply: true,
+    isListening: true,
+    isCaseSensitive: false,
+  },
+  {
+    label: 'Location checking script',
+    prompt: 'Okay, my location is #{location}',
+    handler: function(params) {
+      if (params.location.toLowerCase() === 'nsw' || 
+          params.location.toLowerCase() === 'new south wales') {
+        return 'The most common plants in ' + params.location + 'are ...';
+      } else {
+        return ' I\'m sorry I couldnt find any plants in that area, maybe try the region?';
+        }
+    },
+  },
+  {
+    label: 'Location checking script',
+    prompt: 'My location is #{location}',
+    handler: function(params) {
+      if (params.location.includes(plants.locations)) {
+        return `The most common plant is ${plant.name}...`
+      } else {
+        return ' I\'m sorry I couldnt find any plants in that area, maybe try the region?'
+      }
+    },
+    isListening: true,
+    isCaseSensitive: false,
+  },
```

Comment:
I'm thinking you may have accidentally deleted the ending '];' at the very end, which is why you're receiving an error message.